tests/physics/resolve/ball_ball/test_ball_ball.py

```diff
@@ -90,7 +90,7 @@ def test_translating_head_on_zero_spin_inelastic(
     cb_f, ob_f = model.resolve(cb_i, ob_i, inplace=False)
 
     # Balls should still be moving together in +y direction
-    assert np.isclose(cb_f.vel[1], ob_f.vel[1], atol=1e-10)
+    assert abs(cb_f.vel[1] - ob_f.vel[1]) < 1e-10
```

Comment:
I changed this to a manual floating point comparison so that the pytest output is easier to read if it fails.